packages/core/util/tests/util-specs.js

```diff
@@ -68,6 +122,69 @@ describe('util', () => {
 		});
 	});
 
+	describe('mergeClassNameMaps', () => {
+		const baseMap = {
+			'class-base-only': 'real-class-base-only',
+			'class-shared': 'real-class-shared-base',
+			'class-shared-another': 'real-class-shared-another-base'
+		};
+		const additiveMap = {
+			'class-shared': 'real-class-shared-additive',
+			'class-shared-another': 'real-class-shared-another-additive',
+			'class-additive-only': 'real-class-additive-only'
+		};
+
+		// Do NOT use destructuring operator as 'pxory' does not have properties but have only getters for properties.
+		const avoidProxy = (proxy) => {
```

Comment:
How about 
```suggestion
		// Helper function to get an object from the proxy object that has only getters for properties to make testing easier
		const getResultFromProxy = (proxy) => {
```

Reply:
Look clearer. I will apply it. Thanks for good suggestion.